src/components/ebay-carousel/style-touch.less

```diff
@@ -19,6 +19,12 @@
         }
     }
 
+    &__autoplay {
+        .carousel__list {
```

Comment:
(minor) Can leave out the nesting here

Reply:
I need the nesting here though since `carousel__list` is a child of `carousel__autoplay` (which is on the root element).

Reply:
I misunderstood, added a commit to remove the unneeded nesting. Thanks!